File: CountDiv.cpp
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(int A, int B, int K) {
    // write your code in C++11 (g++ 4.8.2)
    if (A < 0 || B < 0 || K <= 0 || A > B)
    {
        return 0;
    }
    int x = 0;
    int sum = 0;
    if ( A % K == 0 )
    {
        x = 1;
    }
    sum =  B/K - A/K + x;
    return sum;
    
}

File: CountDiv.py
# you can write to stdout for debugging purposes, e.g.
# print "this is a debug message"

def solution(A, B, K):
    # write your code in Python 2.7
    if K <= 0 or A < 0 or B < 0 or A>B:
        return -1
    if (A%K ==0):
        x = 1
    else:
        x = 0
    return B/K - A/K + x